### ISSUE TITLE
Some way to group settings per application?

### ISSUE THREAD

**Issue by XeonG** [keymapperorg/KeyMapper#1231]
Kinda stuck using this app as its the only thing I've found that works for Xiaomi Pad 6 and Pen.. mainly the pen which for some moronic reason the 2 buttons invoke [Page Up] and [Page Down]  which turns out are completely useless in just about every actual app you'd want to use a pen for drawing/painting etc.

So I've found I can assign these buttons to pressing a virtual point onthe screen to select a button certain painting apps with more of a visual interface.. .however switching between these apps is a major pain as the 2x buttons need disabling and renabling per app I'm going to use for different screen points..

is nothing in KeyMapper for grouping settings per application and havine it just switch to them if that is the infocus app, or just being able to go into Keymapper and group them and enable/disable as a group setting instead of manually go through each mapping.

**Comment by jambl3r** [comment]
Yes, try using the 'app in foreground' constraint. You can duplicate your key map and have different constraints for each one, one for each of your apps.
